<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="do_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DIN Pro Cond Medium</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="display">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>570</width>
      <height>427</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DIN Pro Cond Medium</family>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>100</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DIN Pro Cond Medium</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>default</string>
    </property>
    <property name="maxLength">
     <number>12</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DIN Pro Cond Medium</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Данные</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DIN Pro Cond Medium</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ключ</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>40</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DIN Pro Cond Medium</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>default</string>
    </property>
    <property name="maxLength">
     <number>12</number>
    </property>
   </widget>
   <widget class="QPushButton" name="do_3">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>100</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DIN Pro Cond Medium</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="do_4">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>70</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DIN Pro Cond Medium</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Наличие</string>
    </property>
   </widget>
   <widget class="QPushButton" name="do_5">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>190</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DIN Pro Cond Medium</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ReHash</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>190</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DIN Pro Cond Medium</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>170</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DIN Pro Cond Medium</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Новый размер</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
